### ISSUE TITLE
JSTestDriver plugin for Webstorm continues to highlight a browser as attached on restart, eventhough it tells me that "There are no browsers captured"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#173]
```
What steps will reproduce the problem?
1. Start JSTestDriver server and attach a browser
2. Stop the server and close the attached browser/tab


What is the expected output? What do you see instead?
The previously attached browser icon is highlighted, while it should be 
grayed-out/disabled to indicated that the browser is no longer attached.

What version of the product are you using? On what operating system?
I am using WebStorm 2.0 EAP build 98.364 on Mac OSX 10.6.5 with plugin v.1.2.2a


Please provide any additional information below.

```

Original issue reported on code.google.com by `ireney.b...@gmail.com` on 6 Dec 2010 at 10:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 11 Apr 2011 at 7:32
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 17 May 2011 at 5:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This bug is fixed in 1.3.3 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 30 May 2011 at 12:14
- Changed state: **Fixed**
